test/test_api_state_logical_interfaces.py

```diff
@@ -167,6 +175,12 @@ def testLogicalInterfaceActivation(self):
             assert True
             # The expected exception was raised
 
+        try:
+            createdLI.differences()
+            assert False
+        except:
+            assert True
+
```

Comment:
Could you comment this test similarly to the `createdLI.activate()` test immediately above it given that it's not easy to see what the logic is behind this test

Reply:
Okay, I'll comment this and explain it